docs/ensembl-help/ensembl-data/annotation-and-prediction/community-annotation.md

```diff
@@ -0,0 +1,45 @@
+---
+slug: community-genome-annotation
+title: Community genome annotation
+description: Requirements for community annotations to be imported into Ensembl.
+---
+
+# Community annotation formatting
+
+Whilst we encourage our users, submitters and providers to keep submitting their assembly annotations to [INSDC](https://www.insdc.org/about-insdc/), we can understand that this process may take longer than desired, delaying the timelines of when the genome will be available in our site. Thus, we exceptionally accept to get GFF3 files handed over directly to us (although the assembly will still need to be already publicly available in INSDC). We have put together a series of steps that can be easily followed to make sure a GFF3 file is valid and complies with our requirements.
+
+First and most important, the file must be in valid GFF3 format ([specifications](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md)). This can be checked by uploading the file to the [GFF3 Validator](https://genometools.org/cgi-bin/gff3validator.cgi) from “GenomeTools” and clicking on `Validate this file!`. Run it at least once without the `tidy` box ticked to check what the issues are. You can then attempt to validate it again with the `tidy` option checked. Note that if your GFF3 file is larger than the specified limit in the site, you will need to download the tool ([installation instructions](https://github.com/genometools/genometools/blob/master/README.md)) and run it on your computer with the command:
```

Comment:
```suggestion
First and foremost, it is important the file must be in valid GFF3 format ([specifications](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md)). This can be checked by uploading the file to the [GFF3 Validator](https://genometools.org/cgi-bin/gff3validator.cgi) from “GenomeTools” and clicking on `Validate this file!`. Run it at least once without the `tidy` box ticked to check what the issues are. You can then attempt to validate it again with the `tidy` option checked. Note that if your GFF3 file is larger than the specified limit in the site, you will need to download the tool ([installation instructions](https://github.com/genometools/genometools/blob/master/README.md)) and run it on your computer with the command:
```